docs/source/contribute/op_lowering.md

```diff
@@ -184,3 +184,56 @@ You can find the `OpKind` definition in
 [interned_strings.h](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/core/interned_strings.h).
 If the aten symbol is missing, you can submit a PR like
 [this](https://github.com/pytorch/pytorch/pull/36851).
+
+### Debugging op dispatch
+
+Sometimes even after registering the lowering, PyTorch might not make use
```

Comment:
Suggest: "After registering an op lowering, PyTorch might not call it as expected."